hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/exception/S3ErrorTable.java

```diff
@@ -29,6 +31,9 @@
  */
 public final class S3ErrorTable {
 
+  private final static Logger LOG = LoggerFactory.getLogger(
+      S3ErrorTable.class);
```

Comment:
This seems to be the main part matching scope of the title. Should we use separate tickets to track other changes?

Reply:
Sorry, Xiaoyu, I will upload a clean patch.